WordPress/Classes/ViewRelated/Me/App Settings/About/UnifiedAboutViewController.swift

```diff
@@ -59,11 +60,32 @@ class UnifiedAboutViewController: UIViewController {
         ]
     ]
 
+    let headerView: UIView = {
+        // These customizations are temporarily here, but if this VC is moved into a framework we'll need to move them
+        // into the main App.
+        let appInfo = UnifiedAboutHeaderView.AppInfo(
+            icon: UIImage(named: AppIcon.currentOrDefault.imageName) ?? UIImage(),
+            name: (Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as? String) ?? "",
+            version: Bundle.main.detailedVersionNumber() ?? "")
+
+        let fonts = UnifiedAboutHeaderView.Fonts(
```

Comment:
It can be a future enhancement, but it would be good to pass these fonts in so that we can remove the WordPressShared dependency (to make it easier to share the code with the other apps).

Reply:
Sorry, just noticed your comment about this!

Reply:
That's the idea!  Glad to read we're on the same page.

I'm hoping we can make a very clean framework that doesn't have a lot of dependencies, but I'm still not sure how we'll be configuring the whole thing, so right now I'm letting our new VC take care of it.